src/import/pull-common.h

```diff
@@ -7,27 +7,31 @@
 #include "pull-job.h"
 
 typedef enum PullFlags {
-        PULL_FORCE              = 1 << 0,  /* replace existing image */
-        PULL_SETTINGS           = 1 << 1,  /* .nspawn settings file */
-        PULL_ROOTHASH           = 1 << 2,  /* only for raw: .roothash file for verity */
-        PULL_ROOTHASH_SIGNATURE = 1 << 3,  /* only for raw: .roothash.p7s file for verity */
-        PULL_VERITY             = 1 << 4,  /* only for raw: .verity file for verity */
+        PULL_FORCE              = 1 << 0, /* replace existing image */
+        PULL_READ_ONLY          = 1 << 1, /* make generated image read-only */
+        PULL_SETTINGS           = 1 << 1, /* download .nspawn settings file */
+        PULL_ROOTHASH           = 1 << 2, /* only for raw: download .roothash file for verity */
+        PULL_ROOTHASH_SIGNATURE = 1 << 3, /* only for raw: download .roothash.p7s file for verity */
+        PULL_VERITY             = 1 << 4, /* only for raw: download .verity file for verity */
```

Comment:
I know it's existing stuff, but does it make sense to pull the verity file without the root hash or viceversa?

Reply:
roothash without verity definitely makes sense: after all verity might be embedded in the main image (i.e. it's what I think is actually the best approach).

verity without roothash might make sense too, for example, if you have the roothash from some other source already, where it was pinned, and you just need the resources that match it.